CVLByExample/Curve/README.md

```diff
@@ -0,0 +1,28 @@
+
+#Curve 
+
+This directory contains the contract Curve which has the read only reentrancy weakness.
+
+
+##Specs
+certora/specs contains two spec files for exposing this weakness.
+
+1. BuiltinViewReentrancy.spec uses the builtin rule viewReentrancy that checks that for every solidity function and every
```

Comment:
```suggestion
1. `BuiltinViewReentrancy.spec` uses the builtin rule `viewReentrancy` that checks that for every Solidity function and every
```
changes of this style should be applied everywhere, not adding more comments on it

Reply:
done